sfcasts/rector.md

```diff
@@ -1,80 +1,30 @@
-that it has. So down here, I'm going to copy the inside of their static function And
-then paste it over what I have. So basically this sets up rector and kind of points
-at a, a cache file that we'll have that helps us do its job. And it tells it that we
-want to upgrade our code to be Symfony 5.2 compatible, upgrade our code to some
-Symfony code quality standards. If you wanted to know a little bit more about what
-these went, you could open this constant and kind of follow the code. And then
-another rule about constructive injection. I need to look up what that means exactly,
+```terminal
+composer require rector/rector --dev
 
-But of course we don't want to upgrade code is Symfony 5.2 compatible code. We want
-to upgrade all the way to Symfony 5.4. So you might expect me to put, just put five
-at four right here, and I could do that Instead. I'm going to run Symfony level set
-list and then say upgrade to Symfony five, four. Oh, it looks like I also need to add
-a U Sy down here for Symfony set list. Let me retype that and hit tab. Great. What
-the up two Symfony 5.4 means is that it'll include all the updates For, to, to
-upgrade a code of 5.1 5.2 5.3 and 5.4. So instead of listing, needing to list those
-all individually, we can say up two Symfony 5.4, and it will take care of all of
-them. And that's it. So now we're ready to run this before we do. I'm pretty
-interested in what changes this will make. So I'm going to Commit all my changes,
-creating to Symfony 5.4 and installing rector. Perfect. Now to run rector, we're
-going to say vendor bin Rector, and then process. And we can point at a directory to
-process. So I'm going to say source, you could also pass, uh, point it at the config
-and templates directory, but the vast majority of the stuff it's going to do is on
-the source directory
+to install the command line tool. Beautiful! Now, in order for rector to work, it needs a config file and we can boost step one by running rector with:
 
-And it's working And awesome. Eight files have been changed by rec here. Let's scroll
-all the way up to the top here And check out the rule. So this is cool, cause it
-shows you the file that was changed, the change that was made in it. And even below
-kind, it tells you what rules did it apply to get there. So one thing it change is
-user password en coder to user password Hasher. That's A new interface name. It
-changed user, not font exception to username, not font exception to the new user, not
-font exception. So are all just low level things that all just fixes to deprecated
-code that we may or may not have been aware of. And also a change, a change to Ker
-change the use statement there, and then a couple of other similar changes as you go
-along. And also this is part of the coding style, added a re spots return type to all
-of our controllers. So it didn't make a ton of changes, But it did fix a couple of
-our depre,
+```terminal
```

Comment:
This is what I was talking about in the overall comments. "terminal" was still showing up in the formatted .md file for me in VS Code, but it wasn't an issue for the previous terminal entry. Not sure why.

Reply:
Ah ha! I see it! The previous `terminal` block was missing the 3 "ticks" at the end. No biggie though :)